satoshitango/adapter.js

```diff
@@ -0,0 +1,31 @@
+const { Requester, Validator } = require('@chainlink/external-adapter')
+
+const customParams = {
+  base: ['base', 'from', 'coin'],
+  quote: ['quote', 'to', 'market'],
+  endpoint: false
+}
+
+const createRequest = (input, callback) => {
+  const validator = new Validator(callback, input, customParams)
+  const jobRunID = validator.validated.id
+  const endpoint = validator.validated.data.endpoint || 'ticker'
+  const base = validator.validated.data.base.toUpperCase()
+  const quote = validator.validated.data.quote.toUpperCase()
+  const url = `https://api.satoshitango.com/v3/${endpoint}/${quote}`
+
+  const config = {
+    url
+  }
+
+  Requester.request(config)
+    .then(response => {
+      response.data.result = Requester.validateResultNumber(response.data, ['data', 'ticker', base, 'bid'])
```

Comment:
This isn't great, but the API doesn't really offer an actual price.

Reply:
I agree. I've informed Johann about this